polymorpheus.gemspec

```diff
@@ -21,5 +21,4 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency('rake', '~> 10.4.2')
   s.add_development_dependency('rspec', '~> 2.14.0')
-  s.add_development_dependency('mysql2', '~> 0.3.10')
```

Comment:
@jchuang It ends up being roughly the same thing on a new CI run, since the gemfile lock isn't checked into the repo. We'll currently get v0.3.21 either way. That was intentional, since it needs to be a version of mysql2 that is compatible with the corresponding version of ActiveRecord, but it doesn't _need_ to be any specific v0.3.